src/main/java/duke/Task.java

```diff
@@ -0,0 +1,53 @@
+package duke;
+
+/**
+ * Represents a task that is created by an user.
+ */
+class Task {
+    enum Type {
+        E, T, D
+    }
```

Comment:
Maybe consider giving these enums more descriptive names like `EVENT`, `TODO`, and `DEADLINE`, instead of `E`, `T`, `D`.